tests/zero_code_change/test_tensorflow2_gradtape_integration.py

```diff
@@ -97,7 +97,13 @@ def helper_test_keras_v2_gradienttape(script_mode: bool = False, json_file_conte
                 print(log)
                 train_acc_metric.reset_states()
             hook = smd.get_hook()
-            assert not hook
+            assert hook
+            hook.close()
+            # Check that hook created and tensors saved
+            trial = smd.create_trial(path=sim.out_dir)
+            assert len(trial.steps()) > 0, "Nothing saved at any step."
```

Comment:
these might fail on older DLC which dont support ZCC. can you add a check to do this check only if hook is valid?